### ISSUE TITLE
Selection is lost on link creation if SCAYT highlights the word

### ISSUE THREAD

**Issue by jswiderski** [WebSpellChecker/ckeditor-plugin-scayt#90]
This might be a duplicate of https://github.com/WebSpellChecker/ckeditor-plugin-scayt/issues/30
1. Open replacebycode sampe in CKEditor, clear the page and Enable SCAYT.
2. Type any misspelled word, select it and click link button (should be done before SCAYT highlights the word) 
3. Wait a moment.
4. Selection will be lost when SCAYT highlights the word 
5. Type something in input filled and try to insert link - link is either inserted next to word or not inserted at all.

**Comment by wsc-julia-shaptala** [comment]
Hello, this is to notify you that this issue has been fixed on our side and will be included in our next two releases (4.9.3 or 4.9.4)

**Comment by wsc-julia-shaptala** [comment]
Hi @trypton and @jswiderski,

Please be informed that the reported issue has been fixed with the WebSpellChecker release 4.9.4 (Nov 7, 2017).

Sincerely,
